specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/sharedGallery.json

```diff
@@ -532,7 +532,7 @@
           ],
           "x-ms-enum": {
             "name": "HyperVGeneration",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
this will cause enum value type in SDK changed from enum to string. It should be a breaking change for some language.  could you pls share why change this?

Reply:
gallery.json and sharedGallery.json should reuse the same HyperVGeneration object. sharedGallery.json swagger refers to a new feature under development that currently has no customers so I don't believe breaking changes is important at this point for that work.